mobile_verifier/src/heartbeats.rs

```diff
@@ -226,17 +254,6 @@ impl Heartbeat {
         self,
         exec: &mut Transaction<'_, Postgres>,
     ) -> Result<bool, SaveHeartbeatError> {
-        // If the heartbeat is not valid, do not save it
-        if self.validity != proto::HeartbeatValidity::Valid {
-            return Ok(false);
-        }
-
-        sqlx::query("DELETE FROM heartbeats WHERE cbsd_id = $1 AND hotspot_key != $2")
-            .bind(&self.cbsd_id)
-            .bind(&self.hotspot_key)
-            .execute(&mut *exec)
-            .await?;
```

Comment:
is this safe now b/c the query above that pulls valid heartbeats is going to ignore records if the cbsd_id isn't matched to the "current/latest" hotspot_key (so we avoid counting radio heartbeats across multiple gateways?

Reply:
That's not quite right - the current query fetches the latest hotspot key for a cbsd_id and joins that with the previous query, so the table will only return the latest hotspot key